include/exadg/grid/grid.cpp

```diff
@@ -72,18 +72,32 @@ BalancedGranularityPartitionPolicy<dim, spacedim>::partition(
 template<int dim>
 Grid<dim>::Grid(const GridData & data, MPI_Comm const & mpi_comm)
 {
+  typename dealii::parallel::distributed::Triangulation<dim>::Settings distributed_settings;
```

Comment:
if this is only used for parallel::distributed::Triangulation, it should only appear in the corresponding if-branch.

Reply:
moved it to the corresponding if-branch.